src/tsickle.ts

```diff
@@ -2017,6 +2025,41 @@ export function emitWithTsickle(
   };
 }
 
+function addClutzAliases(
+    dtsFileContent: string, sourceFiles: ts.SourceFile[], typeChecker: ts.TypeChecker,
+    host: TsickleHost): string {
+  let reexports = '';
+  for (const sf of sourceFiles!) {
+    const moduleSymbol = typeChecker.getSymbolAtLocation(sf);
+    const moduleExports = moduleSymbol && moduleSymbol.exports || new Map<string, ts.Symbol>();
```

Comment:
why not just an early exit if `!moduleSymbol || !moduleSymbol.exports`. A bit odd to create empty map just to iterate of it.